core/templates/dev/head/pages/exploration_editor/editor_tab/InteractionDetailsCacheService.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/** 
+  * @fileoverview service to provide 
+  * state-specific cache for interaction details. 
+  * It stores customization args
+  * corresponding to an interaction id so that they can be restored if the
+  * interaction is changed back while the user is still in this state. This
+  * cache should be reset each time the state editor is initialized.
+*/
```

Comment:
/** 
 * @fileoverview Service to provide state-specific cache for interaction
 * details. It stores customization args corresponding to an interaction id so
 * that they can be restored if the interaction is changed back while the user
 * is still in this state. This cache should be reset each time the state
 * editor is initialized.
 */
If we write it like this would it cause any linting errors ? (Also notice that the '*/' in the end aligns with all the '*' above)

Reply:
Ok i will align and remove extra line but it does not cause any linting errors . Push command aborts itself if there are any linting errors..

Reply:
Great, so you can maybe format the whole comment like this !